packages/cache/src/internal/tar.ts

```diff
@@ -66,14 +84,24 @@ export async function createTar(
     path.join(archiveFolder, manifestFilename),
     sourceDirectories.join('\n')
   )
+  const workingDirectory = getWorkingDirectory()
+
   // -T#: Compress using # working thread. If # is 0, attempt to detect and use the number of physical CPU cores.
   // --long=#: Enables long distance matching with # bits. Maximum is 30 (1GB) on 32-bit OS and 31 (2GB) on 64-bit.
   // Using 30 here because we also support 32-bit self-hosted runners.
-  const workingDirectory = getWorkingDirectory()
+  // Long range mode is added to zstd in v1.3.2 release, so we will not use --long in older version of zstd.
+  function getProg(): string[] {
```

Comment:
nit: `getProg` isn't very descriptive. Maybe `getCompressionArguments`?

Reply:
I changed it to `getCompressionProgram()`